README.rst

```diff
@@ -25,35 +25,41 @@
-increased concurrency levels.
-
-libEnsemble aims for the following:
-
-• Extreme scaling
-• Resilience/fault tolerance
-• Monitoring/killing of tasks (and recovering resources)
-• Portability and flexibility
-• Exploitation of persistent data/control flow
-
-The user selects or supplies a *generator function* that produces
-input parameters for a *simulator function* that performs and monitors
-simulations. For example, the generator function may contain an
-optimization routine to generate new simulation parameters on-the-fly based on
-the results of previous simulations. Examples and templates of such functions are
-included in the library.
-
-libEnsemble employs a manager/worker scheme that can run on various
-communication media (including MPI, multiprocessing, and TCP); interfacing with
-user-provided executables is also supported. Each worker can
-control and monitor any level of work, from small subnode tasks to huge
-many-node simulations. An executor interface is provided to ensure that scripts
-are portable, resilient, and flexible; it also enables automatic detection of
-the nodes and cores available to the user, and can dynamically assign resources
-to workers.
+libEnsemble is a Python_ toolkit for coordinating asynchronous and dynamic ensembles
+of calculations.
+
+libEnsemble can run on leadership-class machines and facilities,
+and help users take advantage of massively parallel resources to solve design,
+decision, and inference problems and expand the class of problems that can benefit from
+increased parallelism.
+
+libEnsemble's users select or supply **generator** and **simulator** Python
+functions; these respectively produce candidate parameters and perform/monitor
+computations that use those parameters. Generator functions can train
+models, perform optimizations, and test candidate solutions in a batch or streaming
+fashion based on simulation results.
+Simulator functions can themselves use parallel resources and involve libraries
+or executables that are not written in Python.
+With a basic familiarity with Python and NumPy_, users can easily incorporate
+any machine-learning, mathematics, or resource-management libraries into their
+libEnsemble workflows.
+
+libEnsemble aims for:
+
+• **Extreme scaling**, from laptops to supercomputers.
+• **Resilience/fault tolerance**. libEnsemble can restart incomplete tasks or entire ensembles.
+• **Monitoring of applications**, including recovering and dynamically reassigning resources.
+• **Portability and flexibility**. Run identical libEnsemble scripts across machines and facilities.
+• **Exploitation of persistent data/control flow**. libEnsemble can pass data between running functions.
+• **Relatively low start-up cost**. Most single-machine deployments don't require additional services.
+
+libEnsemble employs a manager/worker scheme that runs on MPI, multiprocessing,
+or TCP. Workers control and monitor any level of work using the aforementioned
```

Comment:
What are the "aforementioned functions"?

Reply:
sims and gens. I'll make that more clear

Reply:
Maybe add: Cancellation/killing of simulations.